engine/input/src/input.cpp

```diff
@@ -98,6 +100,16 @@ namespace dmInput
         }
     }
 
+    static GamepadConfig* GetGamepadConfigFromDeviceName(HBinding binding, const uint32_t device_hash)
```

Comment:
Get the gamepad config for the device if it exists. If it doesn't exist the device will be assigned to the unknown gamepad config.